safekit-cluster-mirror/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"OS": {
+			"type": "string",
+			"metadata": {
+				"description": "Operating System to install"
+			},
+			"defaultValue": "2016-Datacenter",
+			"allowedValues": [
+				"2016-Datacenter",
+				"Linux CentOS"
+			]
+		},
+		"vmSize": {
+			"type": "string",
+			"metadata": {
+				"description": "the VM size for all nodes"
+			},
+			"defaultValue": "Standard_A2_v2"
+		},
+		"adminUser": {
+			"type": "string",
+			"metadata": {
+				"description": "User for the Virtual Machines."
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "Password for the Virtual Machines."
+			}
+		},
+		"VIPDnsLabel": {
+			"type": "string",
+			"metadata": {
+				"description": "Public VIP dns label."
+			},
+			"defaultValue": "[concat(resourceGroup().name,'vip')]"
+		},
+		"azurePwsh": {
+			"type": "string",
+			"metadata": {
+				"description": "install azure powershell module (optional)"
+			},
+			"defaultValue": "no",
+			"allowedValues": [
+				"yes",
+				"no"
+			]
+		},
+		"location": {
+			"type": "string",
+			"metadata": {
+				"description": "resources location"
+			},
+			"defaultValue": "westus2",
+			"allowedValues": [
+				"northeurope",
+				"westeurope",
+				"francecentral",
+				"westus2",
+				"centralus"
+			]
+		},
+		"safekitFileUri": {
+			"type": "string",
+			"metadata": {
+				"description": "url of SafeKit package (optional, default to the appropriate Evidian website url)."
+			},
+			"defaultValue": "default"
+		},
+		"_artifactsLocation": {
+			"type": "string",
+			"metadata": {
+				"description": "base URL of deployment resources (template,subtemplates,scripts)"
+			},
+			"defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/safekit-cluster-mirror/"
+		},
+		"_artifactsLocationSasToken": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured."
+			},
+			"defaultValue": ""
+		}
+	},
+	"variables": {
+		"clusternodes": 2,
+		"vmname": "VM1",
+		"moduleName": "mirror",
+		"cltemplate": "[uri(parameters('_artifactsLocation'),'nestedtemplates/cluster.json')]",
+		"ostype": "[if(equals(parameters('OS'), 'Linux CentOS'),'linux','windows')]",
+		"adminUsername": "[if(empty(parameters('adminUser')),concat('admin',resourceGroup().name),parameters('adminUser'))]",
```

Comment:
remove this and use the parameter directly

Reply:
OK